dev/tools/gen_keycodes/lib/logical_key_data.dart

```diff
@@ -387,7 +387,7 @@ class LogicalKeyData {
   }
 
   // Map Web key to the pair of key names
-  static late final Map<String, _ModifierPair> _chromeModifiers = () {
+  static final Map<String, _ModifierPair> _chromeModifiers = () {
```

Comment:
I'm assuming the idea here is that if this is unused, it'll get linted out of existence?

Doesn't this change the initialization time though? We probably lose whatever benefits of lazy initialization from the runtime guards around late checks though...

Reply:
Ahh cool